src/PopoverTooltipItem.js

```diff
@@ -43,7 +43,7 @@ class PopoverTooltipItem extends React.PureComponent<Props> {
 
   render() {
     const label = typeof this.props.label === 'string'
-      ? <Text style={this.props.labelStyle}>{this.props.label}</Text>
+      ? <Text accessibilityLabel={this.props.label.replace(/\n/g, ' ') + '\nToque duas vezes para continuar.'} style={this.props.labelStyle}>{this.props.label}</Text>
```

Comment:
Is there a way to get the `Toque duas vezes para continuar` message from the props instead?

This way, the implementation isn't coupled to a specific language

Reply:
👍